code/game/objects/effects/temporary_visuals/miscellaneous.dm

```diff
@@ -462,3 +462,21 @@
 	else
 		return INITIALIZE_HINT_QDEL
 
+/obj/effect/constructing_effect
+	icon_state = ""
+	density = TRUE
```

Comment:
Are you sure about that?

Reply:
yes

Reply:
wrong!

Reply:
Nice feedback. Do you have a problem with using the default icon state, or making it non passable?

Reply:
It should not be dense

Reply:
@PKPenguin321 why not?

Reply:
If people really don't want it to be dense then I could change it, sure

Reply:
yeah it's a bad idea if you don't want people using the rcd as a portable shield or something

Reply:
What if it had a telefrag effect?

Reply:
You mean building walls over people would kill them? That's not a good idea

Reply:
Its a code issue more than a balance issue really, effects have a lot of weird properties (like becoming permanent when the game MC lags in rare cases) and making a spammable sense-as-a-wall effect is just begging for bugs. Even weirder is that since it's supposed to just be visual it makes no sense to physically impede you

Reply:
well alright uve convinced me

Reply:
FYI, none of these people are maintainers and can be safely ignored.

Reply:
@optimumtact lmao ok ((member))

Reply:
PK had over 200 confirmed kills in club penguin and I've had like one PR merged in the last three months so I ask you: do you feel in charge?